RevenueCat/Plugins/Android/PurchasesWrapper.java

```diff
@@ -320,6 +322,38 @@ public static void setCreative(String creative) {
     public static void collectDeviceIdentifiers() {
         SubscriberAttributesKt.collectDeviceIdentifiers();
     }
+    
+    public static void canMakePayments(String featuresJson) {
+        try {
+            JSONObject request = new JSONObject(featuresJson);
+            JSONArray features = request.getJSONArray("features");
+            List<Integer> featuresToSend = new ArrayList<>();
+            for (int i = 0; i < features.length(); i++) {
+                int feature = features.getInt(i);
+                featuresToSend.add(feature);
+            }
+
+            CommonKt.canMakePayments(UnityPlayer.currentActivity, featuresToSend, new OnResultAny<Boolean>() {
+               @Override
+               public void onReceived(Boolean canMakePayments) {
+                   JSONObject object = new JSONObject();
+                   try {
+                       object.put("canMakePayments", canMakePayments);
+                   } catch (JSONException e) {
+                       e.printStackTrace();
```

Comment:
why the difference in error handling between here and line 324?

Reply:
ah, no real reasoning just copy pasta. i'll update so they're consistent 👍